bin/Main.re

```diff
@@ -31,13 +53,54 @@ let main =
       antialiasing,
       ignoreRegions,
     ) => {
-  module IO1 = (val getIOModule(img1Path));
-  module IO2 = (val getIOModule(img2Path));
+  let img1Type =
+    switch (img1Type) {
+    | `auto when img1 == "_" =>
```

Comment:
I think it should be named stdin instead of auto

Reply:
`auto` is the one which automatically gets the type from the filename (the currently published behaviour). The explicit types (`png`, `jpg`, ...) are the types given with the stdin input. 
I think we can get rid of this though, if we are reading in the format from the magic number.